lib/pr_checker.js

```diff
@@ -9,6 +9,8 @@ const WAIT_TIME_SINGLE_APPROVAL = 24 * 7;
 
 const GITHUB_SUCCESS_CONCLUSIONS = ['SUCCESS', 'NEUTRAL', 'SKIPPED'];
 
+const FAST_TRACK_RE = /^Fast-track has been requested by @(.+?)\. Please 👍 to approve\.$/;
```

Comment:
Would it be possible to loosen this regex so if the wording of the comment changes in the future, it doesn't require to update ncu?

Reply:
I thought about it, but I'm afraid it would match a comment that is not a fast-track request (eg. if someone quotes the fast-track request comment).